customWorldboss.lua

```diff
@@ -819,8 +819,12 @@ local function eS_getEncounterDuration()
 end
 
 local function eS_getTimeSince(time)
-    local dt = GetTimeDiff(time)
-    return dt
+    if time then
+        local dt = GetTimeDiff(time)
+        return dt
+    else
+        return 0
```

Comment:
could you write it in the following way?
```suggestion
    if time then
        return GetTimeDiff(time)
    end

    return 0
```

normally when you use if - else and both return, you don't need to write the else

Reply:
Looks neat, I'll try!

Reply:
if it works, do it also in other part of code

Reply:
I'd rather not spend too much time on a script's code style which is due for a full rewrite anyway.